apache/datadog_checks/apache/apache.py

```diff
@@ -7,6 +7,7 @@
 
 # 3rd party
 import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
```

Comment:
@NoodlesNZ Thanks for creating this PR and apologies it took a long time to review this. Minor comment: since `InsecureRequestWarning` is an exception of `urllib3`, what do you think of doing:
```
from urllib3.exceptions import InsecureRequestWarning
```
... instead?

Reply:
Works for me. I will make the change.